documentation/src/main/asciidoc/userguide/chapters/query/spatial/Spatial.adoc

```diff
@@ -198,29 +198,51 @@ The dialect has been tested with GeoDB version 0.7
 ====
 
 DB2::
-The `DB2SpatialDialect` supports the spatial extensions of the DB2 database. The dialect has been tested with DB2 LUW 11.1
+The `DB2SpatialDialect` supports the spatial extensions of the DB2 LUW database. 
+The dialect has been tested with DB2 LUW 11.1.
+The dialect does not support DB2 for z/OS or DB2 column-oriented databases.
 [NOTE]
 ====
-In DB2 _transfrom groups_ are used to transfer geometries between the database and client applications. Currently Hibernate
-Spatial only supports the ST_WellknownText transfer group.
-====
 
-This dialect can be configured using the Hibernate property:  +
+In order to use the DB2 Hibernate Spatial capabilities, it is necessary to first execute the following
+SQL statements which will allow DB2 to accept Extended WellKnown Text (EWKT) data and return EWKT data.
 
```

Comment:
Could you be more specific here? How should the user execute these statements? Should they be run in a specific schema/database? What privileges are required to be able to execute this?